Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -1338,6 +1338,8 @@ void WebPageProxy::initializeWebPage()
     setDrawingArea(pageClient().createDrawingAreaProxy());
     ASSERT(m_drawingArea);
 
+    websiteDataStore().didInitialize(*this);
```

Comment:
I don't think this is the right place to do this.

Reply:
Indeed — fixed, with Sihui's suggestion above.